packages/cli/generators/observer/index.js

```diff
@@ -3,6 +3,7 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+// no translation: Observer
```

Comment:
How is this comment going to affect the translation process?

On line 24/25, I see:

```
description: g.f('Name of the observer group for ordering'),
```

I think we need to change that message too? For example:

```
description: g.f('Name of the %s group for ordering', 'observer'),